generate_sql.py

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env python3
```

Comment:
What's the idea behind this script - you created the migration instead, right?

Reply:
The commit will be removed later, its the code to generate the migration to keep quoting correct etc.